recipes/fastpfor/all/conanfile.py

```diff
@@ -30,26 +30,37 @@ def config_options(self):
         if self.settings.os == "Windows":
             del self.options.fPIC
 
-    def configure(self):
-        if self.options.shared:
-            self.options.rm_safe("fPIC")
-
     def layout(self):
         cmake_layout(self, src_folder="src")
 
+    def requirements(self):
+        if str(self.settings.arch).startswith("armv8"):
```

Comment:
Why not use `if "arm" in str(self.settings.arch):` or something similar to `if str(self.settings.arch) in [ ...... ] :`? Where are the available architectures for that option defined?

Reply:
@ErniGH
I agree with you.
SIMDe seems to support other arm architectures.
I fixed.